src/main/java/ortus/boxlang/runtime/jdbc/ConnectionManager.java

```diff
@@ -289,7 +320,7 @@ public Connection getConnection( DataSource datasource, String username, String
 	 *
 	 * @return True if the connection was successfully released, otherwise false.
 	 */
-	public boolean releaseConnection( Connection connection ) {
+	public boolean releaseConnection( BoxConnection connection ) {
```

Comment:
We have `getBoxConnection()` and `getUnpooledBoxConnection()`, but no `releaseBoxConnection()`?

I still think there's no need to put `Box` in the `*Connection()` methods. That's just me. :shrug: